hadoop-ozone/dist/src/main/compose/ozonesecure-ha/docker-config

```diff
@@ -54,6 +54,11 @@ OZONE-SITE.XML_ozone.scm.client.address=scm
 OZONE-SITE.XML_hdds.block.token.enabled=true
 OZONE-SITE.XML_hdds.container.token.enabled=true
 OZONE-SITE.XML_hdds.grpc.tls.enabled=true
+
+OZONE-SITE.XML_hdds.x509.default.duration=PT40s
+OZONE-SITE.XML_hdds.x509.renew.grace.duration=PT30s
+OZONE-SITE.XML_hdds.x509.grace.duration.token.checks.enabled=false
```

Comment:
@Galsza now that HDDS-8640 is merged, we can extract these into an add-on config, and the cert-rotation test into an extra shell script.

Reply:
Please remove these.  Currently certificate rotation is not tested in `ozonesecure-ha`.  If you would like to add that, please add them as separate config+script.

Reply:
Thank you, I have removed the unused configs. In the end we are not going to run this test in ha for now